libs/Fft.c

```diff
@@ -223,6 +223,54 @@ FftFontType *FftGetFont(Display *dpy, char *fontname, char *module)
 	return fftf;
 }
 
+#define CACHE_ENTIRES 63
+
+struct {
```

Comment:
```
struct font_cache {
    ....
}
```

... please.

I'd make these static as well.

Reply:
I'm confused about the naming convention though. should we use `struct FontCache` or `struct font_cache`?

It looks like the naming convention is `PascalCase` for all function and structs?

Reply:
Well there is no actual convention (don't forget, this code base is old, so no wonder it's confusing!), but for new things like this, I would like to see `snake_case`.  Same goes for methods as well, ideally.